apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/add-authenticator-modal.tsx

```diff
@@ -249,6 +252,12 @@ export const AddAuthenticatorModal: FunctionComponent<AddAuthenticatorModalProps
      */
     const handleModalSubmit = (): void => {
 
+        selectedAuthenticators.forEach(element => {
+            eventPublisher.publish("application-sign-in-method-add-new-authenticator", {
+                "authenticator": element["defaultAuthenticator"]["name"]
```

Comment:
Null check needed for the primary element?

Reply:
Fixed in #2729